packages/graphql-mapping-template/src/ast.ts

```diff
@@ -388,4 +412,11 @@ export type Expression =
   | CompoundExpressionNode
   | ToJsonNode
   | NotNode
-  | NewLineNode;
+  | NewLineNode
+  | ReturnNode;
+
+/**
+ * Can be used as a parameter to iff() and others to more clearly specify "inline" parameter
+ */
+export const INLINE = true;
```

Comment:
nit: I don't think these are necessary since there are default arguments are used in the functions that use `INLINE` and `RECURSE_INDENT`